01_Data/src/layers/sequelize/model/ChargingStationSecurityInfo.ts

```diff
@@ -0,0 +1,23 @@
+import { Column, Model, Table, DataType } from 'sequelize-typescript';
+import { Namespace } from '@citrineos/base';
+
+/**
+ * Represents the security information found on a particular charging station.
+ */
+@Table
+export class ChargingStationSecurityInfo extends Model {
+  static readonly MODEL_NAME: string = Namespace.ChargingStationSecurityInfo;
+
+  @Column({
+    type: DataType.STRING,
+    unique: true,
+  })
+  stationId!: string;
+
+  // TODO: store public key information into the database
```

Comment:
does it only work with the one charger until this todo is completed?

Reply:
for now, every charger gets the same security config (as needed). so if you have three charging stations and all three needed to send signed meter values, there would be three entries into this table but their public key would be the same 👍

Reply:
aha looks like @thanaParis made the ticket, thank you!! 🙏